app/controllers/api/v1/users/media_controller.rb

```diff
@@ -14,6 +14,16 @@ def update
       render status: :no_content
     end
   end
+
+  def destroy
+    user = User.find(params[:user_id])
+    user_media = user.user_medias.find_by(media_id: params[:id])
+    if user_media
+      user_media.destroy
+    else
+      raise ActiveRecord::RecordNotFound.new "Couldn't find UserMedia with the id: '#{params[:id]}'"
+    end
+  end
```

Comment:
Is there a standard delete status we should be rendering? Maybe just :no_content again?